action.yml

```
@@ -28,18 +37,27 @@ outputs:
 runs:
   using: "composite"
   steps:
+    - id: go-critic
+      run: echo "version=0.8.1" >> $GITHUB_OUTPUT
+      shell: bash
     - id: go-junit-report
       run: echo "version=v2.0.0" >> $GITHUB_OUTPUT
       shell: bash
     - id: gomock
       run: echo "version=v1.6.0" >> $GITHUB_OUTPUT
       shell: bash
+    - id: gosec
+      run: echo "version=v2.16.0" >> $GITHUB_OUTPUT
+      shell: bash
     - id: packer
       run: echo "version=1.8.2" >> $GITHUB_OUTPUT
       shell: bash
     - id: shfmt
       run: echo "version=v3.5.1" >> $GITHUB_OUTPUT
       shell: bash
+    - id: go-staticcheck
+      run: echo "version=v1.3.0" >> $GITHUB_OUTPUT
```

When looking at these tools this was the latest tag available. Is there something I missed?

```suggestion
      run: echo "version=v0.4.3" >> $GITHUB_OUTPUT
```

---

No, it's something I missed. Thanks!